src/vector_v4.hpp

```diff
@@ -109,6 +109,7 @@ template <typename T> class VectorV4 {
 
     // size_
     int size() { return size_; }
+
     int capacity;
```

Comment:
Also, the class member names are inconsistent. You have `size_` and `capacity`. Shouldn't it be `capacity_` ?